spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/oauth/DataflowOAuthIT.java

```diff
@@ -54,12 +56,28 @@ public void testSecuredSetup() throws Exception {
 				.ignoreExceptions()
 				.atMost(120, TimeUnit.SECONDS)
 				.until(() -> {
-					log.debug("Checking auth using curl");
+					log.info("Checking auth using curl");
 					ExecResult cmdResult = execInToolsContainer("curl", "-u", "janne:janne", "http://dataflow:9393/about");
 					String response = cmdResult.getStdout();
 					log.debug("Response is {}", response);
-					boolean ok = response.contains("\"authenticated\":true") && response.contains("\"username\":\"janne\"");
+					Boolean authenticated = JsonPath.parse(response).read("$.securityInfo.authenticated", Boolean.class);
+					String username = JsonPath.parse(response).read("$.securityInfo.username", String.class);
+					boolean ok = Boolean.TRUE.equals(authenticated) && "janne".equals(username);
 					log.info("Check for oauth {}", ok);
+					if (ok) {
+						String version = JsonPath.parse(response).read("$.versionInfo.core.version");
+						log.info("Version=[{}]", version);
+						String api = "tasks/executions";
+						if (VersionUtils.isDataFlowServerVersionGreaterThanOrEqualToRequiredVersion(
+								VersionUtils.getThreePartVersion(version), "2.11.3")) {
+							api = "tasks/thinexecutions";
```

Comment:
I'm not sure why we are testing this endpoint in the DataflowOauthIT test.    It seems like it should be in a different test.   WDYT?

Reply:
Checking some basic security on endpoints. As we add endpoints we can add basic checks to ensure the endpoint security is properly configured.